### PR TITLE
v3.0.21 release

### PR DESCRIPTION
**Pull request by vrundakansara**
- Fix: Avoided redirection for Beaver Builder plugin when activated from import screen.
- Fix: Resolved a few errors with the pre-import PHP compatibility screen.

### CODE REVIEW

File: inc/lib/gutenberg-templates/classes/class-ast-block-templates.php [hunk 1]
```diff
@@ -400,7 +400,7 @@ public function template_assets() {
 						'wpforms_status'          => $this->get_plugin_status( 'wpforms-lite/wpforms.php' ),
 						'gutenberg_status'        => $this->get_plugin_status( 'gutenberg/gutenberg.php' ),
 						'_ajax_nonce'             => wp_create_nonce( 'ast-block-templates-ajax-nonce' ),
-						'button_text'             => esc_html__( 'Starter Templates', 'astra-sites'),
+						'button_text'             => esc_html__( 'Starter Templates', 'ast-block-templates' ),
 						'display_button_logo'     => true,
```
Comment (maheshwaghmare) [inline inc/lib/gutenberg-templates/classes/class-ast-block-templates.php:404]: Textdomain updated.

File: inc/lib/onboarding/classes/class-astra-sites-onboarding-setup.php [hunk 2]
```diff
@@ -0,0 +1,262 @@
+<?php
+/**
+ * Ai site setup
+ *
+ * @since 3.0.0-beta.1
+ * @package Astra Sites
```
Comment (maheshwaghmare) [inline inc/lib/onboarding/classes/class-astra-sites-onboarding-setup.php:6]: Change `3.0.0-beta.1`

File: inc/lib/gutenberg-templates/classes/class-ast-block-templates.php [hunk 3]
```diff
@@ -400,7 +400,7 @@ public function template_assets() {
 						'wpforms_status'          => $this->get_plugin_status( 'wpforms-lite/wpforms.php' ),
 						'gutenberg_status'        => $this->get_plugin_status( 'gutenberg/gutenberg.php' ),
 						'_ajax_nonce'             => wp_create_nonce( 'ast-block-templates-ajax-nonce' ),
-						'button_text'             => esc_html__( 'Starter Templates', 'astra-sites'),
+						'button_text'             => esc_html__( 'Starter Templates', 'ast-block-templates' ),
```
Comment (amithire1201) [inline inc/lib/gutenberg-templates/classes/class-ast-block-templates.php:403]:
```suggestion
						'button_text'             => esc_html__( 'Starter Templates', 'astra-sites' ),
```

File: inc/lib/onboarding/classes/class-astra-sites-onboarding-setup.php [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+				wp_send_json_success( __( 'Bypassing this error.', 'astra-sites' ) );
+			}
+
+			$api_args = array(
+				'timeout'   => 3,
+				'blocking'  => true,
+				'body'      => array(
+					'url'    => esc_url( site_url() ),
+					'err'   => stripslashes( $_POST['error'] ),
+					'id'	=> $_POST['id'],
```
Comment (maheshwaghmare) [inline inc/lib/onboarding/classes/class-astra-sites-onboarding-setup.php:81]: It should be with isset() and with sanitization

**Review by Nikschavan — APPROVED** [review]

File: inc/lib/onboarding/classes/class-astra-sites-onboarding-setup.php (hunk 4 above)
Comment (vrundakansara) [reply inc/lib/onboarding/classes/class-astra-sites-onboarding-setup.php:92]: Done

**Pull request merged**